Clarify leaf node establishment (#1017)
ietf-tapswg/api-drafts: draft-ietf-taps-impl.md

    1.2.2 [2001:DB8::1.80, LTE, TCP]
~~~~~~~~~~

> When an implementation is asked to establish a single connection, only one of the leaf nodes in the candidate set is needed to transfer data. Thus, when a single leaf node becomes ready to use, then the entire connection attempt has been successful. One way to implement this is by having every leaf node update the state of its parent node when it becomes ready, until the trunk node of the tree is ready, which then notifies the application that the connection as a whole is ready to use.

Reviewer: The "entire connection attempt" may still be confusing here. Perhaps better "entire connection establishment" or just "connection establishment"? Or we should replace "connection attempt" in the paragraph that follows, where it refers to an individual node.

Reviewer 2: Perhaps "the connection object becomes ready" would be a clearer alternative here

Author: ```suggestion
When an implementation is asked to establish a single connection, only one of the leaf nodes in the candidate set is needed to transfer data. Thus, once a single leaf node becomes ready to use, then the connection establishment tree is considered ready. One way to implement this is by having every leaf node update the state of its parent node when it becomes ready, until the trunk node of the tree is ready, which then notifies the application that the connection as a whole is ready to use.
```
